tests/installation/partitioning_lvm.pm

```diff
@@ -75,6 +87,48 @@ sub run {
             $collect_logs = 1;
         }
     }
+    elsif (get_var("STORAGE_NG")) {
+        if ($numdisks <= 1) {
+            die "Guided workflow does not skip disk selection when only one disk!"
+                if !match_has_tag('inst-partitioning-scheme');
```

Comment:
Nope, that's why I say that it's matter of taste, so no change is really required. Only point that negated expressions usually are evaluated a little slower, but probably unless does it anyway underneath.